azure-pipelines.yml

```diff
@@ -57,12 +57,20 @@ jobs:
       toolMajorVersion: V2
       suppressionsFile: './scripts/ci/credscan/CredScanSuppressions.json'
       toolVersionV2: '2.1.17'
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@2
```

Comment:
Can this task be run in parallel with CredScan, given CredScan is already the slowest task?

Reply:
Sure, It's just that we need to add one more PostAnalysis task to check the result of the PoliCheck